hub/policy/src/main/java/uk/gov/ida/hub/policy/domain/controller/EidasMatchRequestSentStateController.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.ida.hub.policy.domain.controller;
+
+import uk.gov.ida.hub.policy.PolicyConfiguration;
+import uk.gov.ida.hub.policy.domain.ResponseFromHubFactory;
+import uk.gov.ida.hub.policy.domain.StateTransitionAction;
+import uk.gov.ida.hub.policy.domain.state.EidasMatchRequestSentState;
+import uk.gov.ida.hub.policy.domain.state.EidasSuccessfulMatchState;
+import uk.gov.ida.hub.policy.logging.EventSinkHubEventLogger;
+import uk.gov.ida.hub.policy.services.AttributeQueryService;
+import uk.gov.ida.hub.policy.validators.LevelOfAssuranceValidator;
+
+public abstract class EidasMatchRequestSentStateController<T extends EidasMatchRequestSentState> extends AbstractMatchRequestSentStateController<T, EidasSuccessfulMatchState> {
+
+    public EidasMatchRequestSentStateController(
```

Comment:
Does this constructor do anything? It looks like it just just passes things straight to the parent.
Isn't that the same thing as what would happen if we just didn't write it?

Reply:
You're right, but this a question for the eIDAS team. Since this is work in progress, they might add more properties to the eIDAS states and logic to their controllers. I know they (i.e. Hugh) did it to keep it separate from the non-eidas stuff (for now?)